AI-and-Analytics/Getting-Started-Samples/IntelPython_XGBoost_GettingStarted/README.md

```diff
@@ -36,82 +38,110 @@ You will need to download and install the following toolkits, tools, and compone
 
 Required AI Tools: Intel® Optimization for XGBoost*
```

Comment:
do we plan to rename Intel® Optimization for XGBoost* to XGBoost* everywhere? Then it's  going to be renamed on the Selector page as well